### PR TITLE
Relaunch refactor

### PR DESCRIPTION
**Pull request by xjensen**
This is my attempt at preparing the front-end for relaunch. Here's what's included.

* Sass has been removed.
* The whole component now works without a rebuild.
* Added a way to send language and host-partner data to the API when requesting links (via URL parameters). 
* Removed the `cdn:publish` script. I couldn't find this script in the repo, and anyway, we've got the GitHub Action instead.
* Generally cleaned up and moved around files.
* Removed the EDD test HTML file, because it's no longer working.

I attempted to add the linting from PR #3 (`@open-wc/eslint-config`), but my editor errors out. I'll need to look into that further.

### CODE REVIEW

File: src/index.css.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+export default /* css */ `
```
Comment (xjensen) [inline src/index.css.js:1]: The `/* css */` tag on this template helps with syntax highlighting. There are a few other examples of this. I'm using the following plugin in VSCode.

https://marketplace.visualstudio.com/items?itemName=pushqrdx.inline-html

File: package.json [hunk 2]
```diff
@@ -3,13 +3,13 @@
   "version": "1.1.0",
   "description": "Retrieve and display list of recommended benefits available to Californians",
   "main": "dist/index.js",
+  "type": "module",
   "scripts": {
     "build:js": "esbuild src/index.js --bundle --outfile=dist/index.js --loader:.html=text --loader:.css=text --format=esm",
     "build:more-args": " --minify ",
-    "build": "sass src/css/index.scss src/css/index.css && npm run build:js",
```
Comment (zakiya) [inline package.json:9]: I love sass. :(

Comment (zakiya) [reply package.json:9]: Would having a watch that auto builds and refreshes on change make sass less annoying for you?

File: src/css/index.scss [hunk 3]
```diff
@@ -1,79 +0,0 @@
...
-
-    .details {
-      display: flex;
-      min-width: 69%;
-      align-items: center;
-
-      .svg {
-        margin-top: 0.3rem;
-      }
-
```
Comment (zakiya) [inline src/css/index.scss:50]: I didn't take a close look but I doubt this CSS needs to be this deeply nested. As a rule I try to keep it to 3 or fewer selectors.

**Review by zakiya — APPROVED** [review]
Left some comments. No dealbreakers.

File: src/css/index.scss (hunk 3 above)
Comment (xjensen) [reply src/css/index.scss:50]: Yep, agreed. This is an artifact of the last Sass build. We can fix it at some point.

File: package.json (hunk 2 above)
Comment (xjensen) [reply package.json:9]: I like Sass too!

For a small CSS project like this, where I can see (almost) all of the CSS for the whole project on my screen, I feel the costs of Sass (in build complexity) outweigh the benefits. 

I think Sass (or PostCSS) are more essential for larger projects, where you really need the imports and the scoping from nested selectors, and so on.

**Pull request merged**